_sass/_layout.scss

```diff
@@ -191,3 +191,9 @@
         }
     }
 }
+
+/* Make fragment identifier links visible below the header. */
+:target:before {
+    content: "";
+    display: block;
+    height: 112px; }
```

Comment:
nit: the rest of the file places closing `}` on the next line.

Reply:
Well that's messed up. That's how I had it, but then I noticed the rest of the file it was in was different, so I changed it, then found it was the wrong file, then moved it to this file, and the style is different. -_-